timesheets/Code.gs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+// GLOBAL VARS
+var HRS_START = 4;
+var HRS_END = 8;
+var HRLY_PAY_COLUMN = 9;
+var APPROVAL_COLUMN = 11;
+var EMAIL_COLUMN = 3;
+
+/**
+* Creates the menu item "Run Functions" so user can run scripts on drop-down.
+*/
+function onOpen(e) {
+    var ui = SpreadsheetApp.getUi();
+    ui.createMenu('Run Functions')
+    .addItem('Calculate Pay', 'calculatePay')
+    .addSeparator()
```

Comment:
Remove all the separators. They are useful for creating space between different logical blocks of items, but shouldn't be used between every one.